cpp/src/groupby/hash/groupby.cu

```diff
@@ -57,7 +57,7 @@ constexpr bool array_contains(std::array<T, N> const& haystack, T needle) {
  * @return true `t` is valid for a hash based groupby
  * @return false `t` is invalid for a hash based groupby
  */
-constexpr bool is_hash_aggregation(aggregation::Kind t) {
+bool is_hash_aggregation(aggregation::Kind t) {
   return array_contains(hash_aggregations, t);
```

Comment:
I think adding comments explaining this change will be helpful (this is a temporary fix due to compiler bug and we can resort back to constexpr once cuda 10.2 becomes RAPIDS's minimum compiler version).